src/actions/sessionActions.js

```diff
@@ -2,6 +2,18 @@ import { SubmissionError } from 'redux-form/immutable';
 import { sessionService } from 'redux-react-session';
 
 import sessionApi from '../api/sessionApi';
+import * as types from './actionTypes';
+import history from '../utils/history';
+import routes from '../constants/routesPaths';
+
+export const burnt = () => ({
```

Comment:
Can you name these functions so they explain what they do? Normally with a verb as a prefix, for example, `showToast`, `hideToast`.